integration_test/testgenerated/objects/objects.yml

```diff
@@ -44,6 +44,12 @@ types:
           chan: map<string, string>
       Enum:
         values:
+          - VALUE
```

Comment:
I added `VALUE` and `VALUES` to the test objects to prove these are now handled without collisions.